app/docs/0.10.x/configuration.md

```diff
@@ -203,6 +204,38 @@ http {
 }
 ```
 
+#### Serving API and Website
+
+A common use case for this is to serve a website and API from the
```

Comment:
"for this" implies some context but no such context exist yet because this is the beginning of this chapter.

Maybe better to start like so:
```
A common use case for API providers is to make Kong serve both a website and
the APIs themselves over the Proxy port (`80` or `443` in production). For example:
`https://my-api.com` (website) and `https://my-api.com/api/v1` (API).

To achieve this, we cannot satisfy ourselves with declaring a new HTTP server like the
previous example, and must use a different setup. A good solution is to use a custom
Nginx configuration template which adds a new `location` block serving the website
alongside the Kong Proxy `location` block.

Example:
```